InventoryInStorePickup/Model/SearchRequest/Builder/AreaBuilder.php

```diff
@@ -16,10 +16,7 @@
 class AreaBuilder
```

Comment:
It seems to be a part of API.
Probably it should be moved to API or at least be marked as `@api?`
See `\Magento\Framework\Api\AbstractSimpleObjectBuilder`, `\Magento\Framework\Api\SearchCriteria`, '\Magento\Framework\Api\SearchCriteriaBuilder'.

I believe it should be discussed.

Reply:
The API only is \Magento\InventoryInStorePickupApi\Model\SearchRequestBuilderInterface
All other builders - implementation.